sig-operations/charter.md

```diff
@@ -0,0 +1,37 @@
+# SIG Operations
+
+## Scope
+
+The goal of this SIG is to define and coordinate management and operations for hardware and platform tooling within the
```

Comment:
Would like to see more specific here, whether this SIG is intended to cover all varients of community clouds or is specific to the initial one that is OpenStack on bare metal based.

Reply:
I think being specific would just limit us upfront. This sig is responsible for all the platform component, including OpenStack, if we have some in the future...